src/monitor-control-service/azext_amcs/manual/_params.py

```diff
@@ -30,6 +32,14 @@
     KnownWindowsEventLogDataSourceStreams, KnownSyslogDataSourceStreams, \
     KnownSyslogDataSourceFacilityNames, KnownSyslogDataSourceLogLevels
 
+logger = get_logger(__name__)
+
+
+def validate_association_name_with_endpoint(cmd, namespace):
+    if namespace.endpoint_id:
+        if namespace.association_name != "configurationAccessEndpoint":
+            raise CLIError("Association name for resource to endpoint must be configurationAccessEndpoint")
```

Comment:
CLIError is not recommended to use. Please follow this [doc](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md)